go-controller/pkg/ovn/endpoints.go

```diff
@@ -233,13 +237,23 @@ func (ovn *Controller) deleteEndpoints(ep *kapi.Endpoints) error {
 				lb, err)
 			continue
 		}
-		key := fmt.Sprintf("\"%s:%d\"", svc.Spec.ClusterIP, svcPort.Port)
-		_, stderr, err := util.RunOVNNbctl("remove", "load_balancer", lb,
-			"vips", key)
-		if err != nil {
-			logrus.Errorf("Error in deleting endpoints for lb %s, "+
-				"stderr: %q (%v)", lb, stderr, err)
+
```

Comment:
`quotedHostPort := '"' + net.JoinHostPort(svc.Spec.ClusterIP, svcPort.Port) + '"'` and use that in the if/else branches below.

Reply:
Done